src/class.c

```diff
@@ -584,8 +584,8 @@ void mrb_hash_check_kdict(mrb_state *mrb, mrb_value self);
     S:      String         [mrb_value]            when ! follows, the value may be nil
     A:      Array          [mrb_value]            when ! follows, the value may be nil
     H:      Hash           [mrb_value]            when ! follows, the value may be nil
-    s:      String         [char*,mrb_int]        Receive two arguments; s! gives (NULL,0) for nil
-    z:      String         [char*]                NUL terminated string; z! gives NULL for nil
+    s:      String         [char const*,mrb_int]  Receive two arguments; s! gives (NULL,0) for nil
+    z:      String         [char const*]          NUL terminated string; z! gives NULL for nil
```

Comment:
L775 uses `const char*` and here it's `char const*`. Aren't they inconsistent?

Reply:
Since `"d"` is `TYPENAME const`, I intended to match it.
C/C++ recognizes that it is equivalent to `const TYPENAME`, so there is no problem.
However, in addition to that, human beings are certainly confused when the pointer is involved, so I will correct it.